scripts/url.js

```diff
@@ -31,7 +32,7 @@ const shortenUrls = async () => {
       return o;
     }, {});
 
-  fs.writeFileSync("urls.json", `${JSON.stringify(sorted, null, 2)}\n`);
+  writeFileSync("urls.js", `export default ${JSON.stringify(sorted, null, 2)}\n`);
```

Comment:
Note: It's not as easy to import JSON files in ESM, so I switch urls.json to urls.js.

Reply:
interesting — thanks!